code/languages/com.mbeddr.mpsutil/solutions/com.mbeddr.mpsutil.process.sandbox/models/com/mbeddr/mpsutil/process/sandbox/plugin.mps

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<model ref="r:9aef0d3c-87c9-4ce3-ad67-def9817c1ab4(com.mbeddr.mpsutil.wizard.sandbox.plugin)">
```

Comment:
Changes of language name in the references from `c.m.mpsutil.wizard` to `c.m.mpsutil.process` looks wrong - the language has been previously renamed `c.m.mpsutil.wizard` (see also the change log).

Reply:
looks like the modules.xml got outdated. I've added back the missing solution and the deps. should be updated now as well.